sn_faucet/src/faucet_server.rs

```diff
@@ -11,17 +11,24 @@ use color_eyre::eyre::{eyre, Result};
 use serde::{Deserialize, Serialize};
 use sn_client::Client;
 use sn_transfers::{LocalWallet, NanoTokens};
+use std::collections::HashMap;
 use std::path::{self, Path, PathBuf};
+use std::str::FromStr;
 use tiny_http::{Response, Server};
 use tracing::{debug, error, info, trace};
 
 const SNAPSHOT_FILENAME: &str = "snapshot.json";
 const SNAPSHOT_URL: &str = "https://api.omniexplorer.info/ask.aspx?api=getpropertybalances&prop=3";
+const PUBKEYS_URL: &str = "https://pastebin.com/raw/pUm6tVRN";
```

Comment:
Instead of loading the keys from a pastebin link, wouldn't it be safer to have those in a json/csv file locally in the repo?

Reply:
Oh I like that idea! Before I commit this, just a quick sanity check on the size of the file and whether it's not too big for the repo... 155KB for 1227 addresses is an average of 126B per address. For about 16,000 addresses holding maid that would be a file about 2MB. Is that acceptable to have committed in the repo? It seems ok to me but wanted to check first.

Reply:
Very acceptable 👍

Reply:
Added this change in https://github.com/maidsafe/safe_network/pull/1210/commits/779a39db98acce7f0e01b90b8f05bc58b2e026d0 and also allowed the request to fail without preventing the faucet to start (ie http status 404 allows the process to continue, not fail)